lib/apb.ts

```diff
@@ -82,12 +87,13 @@ export class apb {
 
   //* BOOLEAN CHECKS & Validators /////////////////////////////////////////////////////
 
-  isStateIntegration(stateName: string, States = this.States) {
+  isInteractionStateWithParameters(stateName: string, States = this.States) {
     if (States[stateName] === undefined) {
       throw new Error(`State ${stateName} does not exist in the States object`)
     }
-    const state_to_check : State | TaskState | InteractionState = States[stateName]
-    return ((state_to_check.Type === "Task" || state_to_check.Type === "Interaction") && !!state_to_check['Parameters'])
+    const state_to_check : State | InteractionState = States[stateName]
+    // return ((state_to_check.Type === "Task" || state_to_check.Type === "Interaction") && !!state_to_check['Parameters'])
+    return (state_to_check.Type === "Interaction" && !!state_to_check['Parameters'])
```

Comment:
question: This transformed from an Integration check to an InteractionState check. Is that correct? Are checks for Integration states till happening correctly elswhere in the code? Or is that not needed anymore?

Reply:
[As you can see in this code before the removal of helper states](https://github.com/twilio-labs/sls-apb/blob/3f10714b33336cd29891d5b3cff84dab8e857e51/lib/apb.ts), `isStateIntegration` was only used to:
1. check if a task needed a helper state
2. check if an Interaction needed a helper state
3. correctly resolve a step name (because some steps had helper states)

this code could probably be removed entirely